### PR TITLE
update myalgo packages

### PR DESCRIPTION
**Pull request by thdailong**
Closes #x

## Proposed Changes

- Update myalgo wallet for support signing multisig transaction

## TODO
- [x] update myalgo packages

### CODE REVIEW

File: packages/web/src/lib/myalgowallet-mode.ts [hunk 1]
```diff
@@ -29,17 +30,21 @@ interface MyAlgoConnect {
...
 	/**
 	 * @async
 	 * @description Sign an Algorand Transaction.
 	 * @param transaction Expect a valid Algorand transaction array.
+	 * @param signOptions Sign transactions options object.
 	 * @returns Returns signed an array of signed transactions.
 	 */
-	signTransaction(transaction: (AlgorandTxn | EncodedTransaction)[]): Promise<SignedTx[]>;
+	signTransaction(transaction: (AlgorandTxn | EncodedTransaction)[], 
+		signOptions?: SignTransactionOptions): Promise<SignedTx[]>;
```
Comment (robert-zaremba) [inline packages/web/src/lib/myalgowallet-mode.ts:47]: why do we have two times the same function? (line 36 and line 47)?

File: yarn.lock [hunk 2]
```diff
@@ -99,7 +99,7 @@ __metadata:
   resolution: "@algo-builder/web@workspace:packages/web"
   dependencies:
     "@json-rpc-tools/utils": ^1.7.6
-    "@randlabs/myalgo-connect": ^1.1.2
+    "@randlabs/myalgo-connect": ^1.1.3
```
Comment (robert-zaremba) [inline yarn.lock:102]:
```suggestion
    "@randlabs/myalgo-connect": ^1.2.0
```

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Please upgrade myalgo-connect to v1.2.0 and see if we can remove duplicated function.

File: packages/web/src/lib/myalgowallet-mode.ts (hunk 1 above)
Comment (thdailong) [reply packages/web/src/lib/myalgowallet-mode.ts:47]: Line 36 is the `signTransaction` for signing a transaction but in Line 47,  `signTransaction` for signing an array of transaction

**Pull request merged**